internal/cloud/realm/error.go

```diff
@@ -3,7 +3,15 @@ package realm
 import (
 	"bytes"
 	"encoding/json"
+	"errors"
 	"net/http"
+
+	"github.com/10gen/realm-cli/internal/utils/api"
+)
+
+const (
+	// InvalidSessionCode is the error code returned when the user's sesison is invalid
```

Comment:
docstring can be removed for an unexported const (i'd remove, but if you want to keep it then do some cleanup)

```go
// invalidSessionCode is the server error code for an invalid session
```